src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritManagement/index.jelly

```diff
@@ -1,3 +1,4 @@
+<?jelly escape-by-default='true'?>
```

Comment:
That is fine. You only need to look for cases where the view _intends_ to display HTML content bound to some model object attribute (`something.detailMessage`, etc.), where the content is known be untainted by user input. Adding `escape-by-default='true'` without switching that usage to `j:out` will cause the content to be rendered as plain text rather than markup, breaking the UI.